# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wasi-libcxx/hotfix-abi-libcxx.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./libcxxabi/src/CMakeLists.txt.vanilla	2024-11-07 15:42:08.260140021 +0100
+++ ./libcxxabi/src/CMakeLists.txt	2024-11-07 15:42:20.090140209 +0100
@@ -314,7 +314,7 @@
 
 # Add a meta-target for both libraries.
 add_custom_target(cxxabi DEPENDS ${LIBCXXABI_BUILD_TARGETS})
-add_dependencies(cxxabi-test-depends cxxabi cxx)
+#add_dependencies(cxxabi-test-depends cxxabi cxx)
 
 if (LIBCXXABI_INSTALL_LIBRARY)
   install(TARGETS ${LIBCXXABI_INSTALL_TARGETS}
